docs/how-to-guides/new-user-guides/manage-clusters/create-kubernetes-persistent-storage/manage-persistent-storage/install-iscsi-volumes.md

```diff
@@ -31,3 +31,19 @@ services:
       - "/etc/iscsi:/etc/iscsi"
       - "/sbin/iscsiadm:/sbin/iscsiadm"
 ```
+
+You need to mount the file `/usr/lib64/libcrypto.so.10` if the platform is RHEL 7.9, as shown in the sample below.
```

Comment:
```suggestion
If you're using RHEL 7.9, you need to mount the file `/usr/lib64/libcrypto.so.10` as shown in the sample below.

```

Reply:
@jiaqiluo https://github.com/rancher/rancher/issues/41591 only mentions RHEL 9.0 as not affected, do you know if this is an issue with 8.x since that's also listed in the support matrix?

Reply:
@btat Good call!   I tested RHEL 8.6 and found it is affected by another similar file. I have added it to the PR.